public/pages/DetectorConfig/containers/DetectorConfig.tsx

```diff
@@ -21,14 +21,15 @@ import { RouteComponentProps } from 'react-router';
 import { AppState } from '../../../redux/reducers';
 import { useSelector, useDispatch } from 'react-redux';
 import { getDetector } from '../../../redux/reducers/ad';
+import { EuiLoadingSpinner } from '@elastic/eui';
 
 interface DetectorConfigProps extends RouteComponentProps {
   detectorId: string;
   onEditFeatures(): void;
   onEditDetector(): void;
 }
 
-export const DetectorConfig = (props: DetectorConfigProps) => {
+export function DetectorConfig(props: DetectorConfigProps) {
```

Comment:
just curious: why this change is needed?

Reply:
This is to keep consistent with other places.